x-pack/plugins/maps/public/classes/sources/vector_source/vector_source.tsx

```diff
@@ -179,6 +180,10 @@ export class AbstractVectorSource extends AbstractSource implements IVectorSourc
     throw new Error('Should implement VectorSource#deleteFeature');
   }
 
+  async getSourceIndexList(): Promise<string[]> {
```

Comment:
getSourceIndexList is only used internally in es_search_source. There is no need to add it to IVectorSource interface.